x-pack/plugins/apm/common/__snapshots__/apm_telemetry.test.ts.snap

```diff
@@ -524,6 +524,710 @@ exports[`APM telemetry helpers getApmTelemetry generates a JSON object with the
                               }
                             }
                           }
+                        },
```

Comment:
@elastic/telemetry this diff shows the new fields needed to be added to the mapping.

Reply:
I'd love we get to eventually integrate this collector to the `schema` convention (https://github.com/elastic/kibana/issues/72736). This week I'll be working on some `schema` improvements. Hopefully, I'll get there so we don't need to maintain this snapshot.

@mindbat FYI about APM added fields

Reply:
Thanks for the ping, @afharo!

@smith Once these changes are ready, would you mind opening a [mapping update issue](https://github.com/elastic/infra/issues/new?assignees=&labels=area%3Atelemetry&template=telemetry-mapping-update.md&title=Request+to+update+Telemetry+mapping+for+%3CFIELDS%2FINDICES%3E), in the Infra repo (with a list of the new fields), so we can track the changes needed for these to show up in the indexed data?

Reply:
Opened elastic/telemetry#461.

Reply:
@smith Ah, those changes need to go to the Infra repo, not telemetry (see my link above).

We've (Infra) taken over the maintenance of the telemetry cluster, so we'll be making these changes 😉